src/main/java/com/google/cloud/tools/libraries/libraries.json

```diff
@@ -1,5 +1,105 @@
 [
   {
+    "name": "Google Cloud Vision",
+    "documentation": "https://cloud.google.com/vision/docs/reference/libraries",
+    "description":"The Google Cloud Vision API provides vision features including image labeling, face and landmark detection, optical character recognition (OCR), and tagging of explicit content.",
+    "transports": ["grpc"],
+    "icon":"https://cloud.google.com/_static/images/cloud/products/logos/svg/vision.svg",
+    "clients": [
+      {
+      "name":"Google Java Client for Cloud Natural vision",
+      "vision":"java",
+      "site":"https://cloud.google.com/vision/docs/reference/libraries#client-libraries-install-java",
+      "apireference":"https://googlecloudplatform.github.io/google-cloud-java/latest/apidocs/com/google/cloud/vision/package-summary.html",
+      "infotip":"The com.google.cloud.vision.v1 packages",
+      "launchStage" : "beta",  
+      "packages" : ["com.google.cloud.vision.v1"],
+      "source" : "https://github.com/GoogleCloudPlatform/google-cloud-java/tree/master/google-cloud-vision",
+      "visionLevel" : "1.7.0",
+      "mavenCoordinates" : {
+         "groupId" : "com.google.cloud",
+         "artifactId" : "google-cloud-vision",
+         "version" : "0.22.0-beta"
+       }
+      }
+    ]
+  },
+  {
+    "name": "Google Cloud Natural Language",
+    "documentation": "https://cloud.google.com/language/docs/reference/libraries",
+    "description":"The Google Cloud Natural Language API provides natural language understanding technologies including sentiment analysis, entity analysis, and syntax analysis.",
+    "transports": ["grpc"],
+    "icon":"https://cloud.google.com/_static/images/cloud/products/logos/svg/language.svg",
+    "clients": [
+      {
+      "name":"Google Java Client for Cloud Natural Language",
```

Comment:
Ditto _Java Client_

Reply:
Java Client Libraries for the Google Cloud Natural Language API